packages/www/components/FeaturesModel/index.tsx

```diff
@@ -45,7 +45,9 @@ export default function FeaturesModel() {
 
   return (
     <AlertDialog open={shouldShowFeature}>
-      <AlertDialogContent css={{ p: "0", borderRadius: 0 }}>
+      <AlertDialogContent
+        className="bg-surface"
```

Comment:
This should be styled at the component level, not at every Alert Dialog level. We don't want to have to apply this class to *every* dialog, the underlying dialog should be responsible for that.

Reply:
Same here... once we create the Alert from scratch, all the styling will be inside the component level